README.md

```diff
@@ -85,7 +85,6 @@ Custom BPMN Theme features will be progressively added to `bpmn-visualization`.
 - [Custom user task icon](examples/custom-bpmn-theme/custom-user-task-icon/README.md) - use your own icon
 - [Custom colors](examples/custom-bpmn-theme/custom-colors/README.md) - custom defaults, specific to BPMN element types
 - [Custom fonts](examples/custom-bpmn-theme/custom-fonts/README.md) - custom defaults, specific to BPMN element types
-- [Hacktoberfest themes](examples/custom-bpmn-theme/hacktoberfest-diagram/README.md) - special Hacktoberfest diagram with Hacktoberfest colors
```

Comment:
we didn't do it for the monitoring demo, but we could add a dedicated section for all demos as we now have readme for some of them.
In the repo readme, we could add just a link to a new readme in the demo folder that will ref the demo

Reply:
Done